docs/docs/hpc-on-k8s/user.md

```diff
@@ -0,0 +1,129 @@
+# Steps to create bio-user in bio namespace
+
+Apply the YAML file for `bio-user`:
+
+```bash
+# Creating physics namespace
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: bio
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: bio-user
+    namespace: bio
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+    namespace: bio
+    name: bio-role
+rules:
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs: ["*"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+    namespace: bio
+    name: bio-rolebinding
+subjects:
+  - kind: ServiceAccount
+    name: bio-user
+    namespace: bio
+roleRef:
+    kind: Role
+    name: bio-role
+    apiGroup: rbac.authorization.k8s.io
+```
+
+Create `bio-user` by applying the below command. 
+
+```bash
+kubectl apply -f bio-user.yaml
+```
+
+## Add configuration to `~/.kube/config`
+
+```bash
+kubectl config set-credentials bio-user --token=$(kubectl get secret -n bio $(kubectl get sa bio-user -n bio -o jsonpath='{.secrets[0].name}') -o jsonpath='{.data.token}' | base64 --decode)
+kubectl config set-context bio-context --cluster=$(kubectl config view -o jsonpath='{.clusters[0].name}') --namespace=bio --user=bio-user
+```
+
+
+## Switch to bio-context
+
+```bash
+kubectl config use-context bio-context
+```
+
+## Steps to create physics-user in bio namespace
```

Comment:
This step being a H2 puts it at the same level as the rest of the docs. I have tripped up trying to add this user because I've followed the doc in order and switched to the `bio-context` and then received errors.

Fixing the formatting and probably noting the differences between the different contexts (as mentioned in the comment earlier in this file) should clear this up

Reply:
updated thanks